Nfiann-arguments (#7128)
fishtown-analytics/docs.getdbt.com: website/docs/reference/global-configs/behavior-changes.md

When you set the `validate_macro_args` flag to `True`, dbt will:
- Check that all argument names in your YAML match those in the macro definition
- Raise warnings if the names or types don't match

> - Validate that the [`type` values follow the supported format](/reference/resource-properties/arguments#supported-types-for-macro-argument-validation).

Reviewer: ```suggestion
- Validate that the [`type` values follow the supported format](/reference/resource-properties/arguments#supported-types).
```

> - If you use this flag and no arguments are documented in the YAML, dbt will infer them from the macro and include them in the [`manifest.json` file](/reference/artifacts/manifest-json)

Reviewer 2: ```suggestion
- If no arguments are documented in the YAML, infer them from the macro and include them in the [`manifest.json` file](/reference/artifacts/manifest-json)
```

redundant of sentence leading into bulleted list "When you set the `validate_macro_args` flag to `True`, dbt will..."

- Raise warnings if the names or types don't match

> - Validate that the `types` values follow the supported format explained in the next section

Reviewer 3: since we moved the 'supported types' from this page to the `type` doc, we need to update this and add a link to the supported types i think
```suggestion
- Validate that the [`type` values follow the supported format](/reference/resource-properties/arguments#supported-types-for-macro-argument-validation).
```
